<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrjSettingsDialog</class>
 <widget class="QDialog" name="PrjSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="95,5">
     <item>
      <widget class="QTabWidget" name="tabs">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="projectTab">
        <attribute name="title">
         <string>Project</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTableWidget" name="projectProperties">
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <row>
            <property name="text">
             <string>Project name</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Project path</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>Property</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Value</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string>Project name:</string>
            </property>
           </item>
           <item row="0" column="1">
            <property name="text">
             <string/>
            </property>
           </item>
           <item row="1" column="0">
            <property name="text">
             <string>Project path:</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="playerTab">
        <attribute name="title">
         <string>Player</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTableWidget" name="playerProperties">
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <row>
            <property name="text">
             <string>Start layer</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Start tile</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Speed</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>View distance</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>Property</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Value</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string>Spawn layer:</string>
            </property>
           </item>
           <item row="1" column="0">
            <property name="text">
             <string>Spawn tile:</string>
            </property>
           </item>
           <item row="2" column="0">
            <property name="text">
             <string>Speed:</string>
            </property>
           </item>
           <item row="3" column="0">
            <property name="text">
             <string>View distance:</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="bSave">
         <property name="text">
          <string>Save and Close</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="bCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
